First try of a stop RStudio IT snippet (#1690)
galaxyproject/training-material: snippets/stop_RStudio.md

> ### {% icon hands_on %} Hands-on: Stop RStudio
>

> > If you want to stop your RStudio interactive tool, once you finish your R treatment and if relevant, saving your RStudio environment in your Galay history, you can proceed in 2 different ways:

Reviewer: let's document here also how to save datasets, just in case. This snippet definitely needs to be used in some places :)

```suggestion
> When you have finished your R analysis, it's time to stop RStudio. 
>
> First, save your work into Galaxy, to ensure reproducibility:
> 1. You can use `gx_put(filename)` to save individual files by supplying the filename
> 2. You can use `gx_save()` to save the entire analysis transcript and any data objects loaded into your environment.
>
> Once you have saved your data, you can proceed in 2 different ways:
```

> > > Deleting the corresponding history dataset named `RStudio` and showing a "in progress state", so yellow.

Reviewer: ```suggestion
> > - Deleting the corresponding history dataset named `RStudio` and showing a "in progress state", so yellow, OR
```

> > > Clicking on the "User" menu at the top and go to "Active InteractiveTools" and locate the RStudio instance you started, ticking the corresponding box so you can finally click on the "Stop" button.

Reviewer: ```suggestion
> > - Clicking on the "User" menu at the top and go to "Active InteractiveTools" and locate the RStudio instance you started, selecting the corresponding box, and finally clicking on the "Stop" button at the bottom.
```
